tuxbot/main.go

```diff
@@ -7,13 +7,42 @@ import (
 	"bytes"
 	"log"
 	"os"
+	"os/exec"
 
 	"github.com/keybase/slackbot"
 	"github.com/keybase/slackbot/cli"
+	"github.com/nlopes/slack"
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
-func kingpinTuxbotHandler(args []string) (string, error) {
+var token string
```

Comment:
Part of me thinks this is fine, part of me doesn't like globals...

We could make a function `func slackToken() string` that returns `os.Getenv("SLACK...` instead of global?